packages/extension-api/src/extension-api.d.ts

```diff
@@ -17,15 +17,21 @@
  ***********************************************************************/
 
 /**
- * The Podman Desktop API provides a way to interact with Podman Desktop.
+ * The Podman Desktop API provides a way to build extensions interacting with Podman Desktop.
```

Comment:
it does not provides a way to build extensions, the extensions consume that API.